spec/amp-var-substitutions.md

```diff
@@ -245,6 +246,15 @@ The tables below list the available URL variables grouped by type of usage. Furt
 | [Timer Duration](#timer-duration) | N/A | `${timerDuration}` |
 | [Timer Start Time](#timer-start) | N/A | `${timerStart}` |
 
+### AMP Components
+| Variable Name | Platform Variable | amp-analytics Variable | Required AMP Components
+|---------------|-------------------|------------------------|-----------------------|
+| [Variant](#experiment-variant) | VARIANT | N/A | [`<amp-experiment>`](https://github.com/ampproject/amphtml/blob/master/extensions/amp-experiment/amp-experiment.md) |
+| [Variants](#experiment-variants) | VARIANTS | N/A | [`<amp-experiment>`] |(https://github.com/ampproject/amphtml/blob/master/extensions/amp-experiment/amp-experiment.md)
```

Comment:
The link is outside the table on this one, can we move it in?

Reply:
oops fixed : )